src/components/Table.tsx

```diff
@@ -46,16 +46,20 @@ export const TableCell = styled.div<{ align?: 'center' | 'left' | 'right' }>`
     }
 `;
 
-export const TableRow = styled.div<{ focused?: boolean }>`
+export const TableRow = styled.div<{ focused?: boolean; disable?: boolean }>`
     display: contents;
-    cursor: pointer;
 
     color: ${textColor};
     text-decoration: none;
 
-    &:hover ${TableCell} {
-        background-color: ${gray6};
-    }
+    ${({ disable }) =>
+        !disable &&
+        css`
```

Comment:
you don't need css here probably
https://github.com/taskany-inc/bricks/blob/main/src/components/MenuItem.tsx#L44